src/event_loop.rs

```diff
@@ -490,6 +491,18 @@ impl ActiveEventLoop {
             platform: self.p.owned_display_handle(),
         }
     }
+
+    /// Returns the current cursor position relative to the top-left hand corner of the desktop.
```

Comment:
I think we've agreed that it should be the same as in events, @daxpedda ? 

So it should be relative to Window itself.

If it's not, it should be a separate API call with a special name.

Reply:
Oh, nvm, it's on the `ActiveEventLoop`.

I'll look once I have time then.

Reply:
Yeah, it wouldn't make sense indeed.